src/arrayMethodSort.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => String(a) > String(b),
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = 0; j < this.length; j++) {
+        if (compareFunction(this[j], this[j + 1]) > 0) {
```

Comment:
The condition in the if statement should check for `compareFunction(this[j], this[j + 1]) > 0` to be consistent with the default Array.sort() behavior, where the function should return a positive number to indicate that `this[j]` should come after `this[j + 1]`.